File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1181.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
 
        int n = ler.nextInt();
        String e = ler.next();
        double soma = 0, valor;
        int i, j;
        for(i=0;i<12;i++){
            for(j=0;j<12;j++){
                valor = ler.nextDouble();
                if(i==n){
                    soma+=valor;
                }
            }
        }
        
        if(e.equals("S")){
            System.out.printf("%.1f\n", soma);
        }else{
            System.out.printf("%.1f\n", soma/12);
        } 
        
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1181.lua
n = io.read("*number")
e = io.read()
e = io.read()
soma=0.0
for i=0,11 do
    for j=0,11 do
        valor = io.read("*number")
        if i==n then soma=soma+valor end
    end
end
if e=='S' then print(soma)
else print(string.format("%.1f", soma/12)) end
